zerver/tests/test_openapi.py

```diff
@@ -874,7 +875,8 @@ def test_generate_and_render_curl_example_with_excludes(self) -> None:
             "    --data-urlencode num_before=4 \\",
             "    --data-urlencode num_after=8 \\",
             '    --data-urlencode \'narrow=[{"operand": "Denmark", "operator": "channel"}]\' \\',
-            "    --data-urlencode use_first_unread_anchor=true",
+            "    --data-urlencode use_first_unread_anchor=true \\",
+            "    --data-urlencode 'message_ids=[1, 2, 3]'",
```

Comment:
Generally, we want these tests to be sample of normal requests to make, and I think that means we don't want to add this new rare-use parameter here. Also this reveals that `use_first_unread_anchor` needs to be incompatible with the new parameter. 

(You might also want a prep commit to rework these examples to use the modern API; `use_first_unread_anchor` is deprecated, right?)

Reply:
Added a prep commit to remove usage of `use_first_unread_anchor` and added `message_ids` to the excluded parameteres list.